x-pack/plugins/enterprise_search/server/routes/app_search/crawler.ts

```diff
@@ -57,14 +59,14 @@ export function registerCrawlerRoutes({
   );
 
   router.post(
-    {
+    skipBodyValidation({
```

Comment:
question: Why are we skipping body validation here?

Reply:
This lets us pass arbitrary props to the BE, so we don't have to worry about keeping these up to date with our endpoints.

Reply:
Right, is that something we want to be doing? I'm assuming we're doing the validation for a good reason.

Reply:
Validation is implemented/enforced by the Kibana API proxy, so if you don't define a prop in the body it won't pass it back to enterprise search.  I don't find it useful personally haha.   Let me send my data!